content/blocks/lunch/index.md

```diff
@@ -4,4 +4,5 @@ headless="true"
 time=60
 +++
 
-Every Saturday at CYF we cook and eat together. We share our food and our stories. We learn about each other and about the world. We build community.
+Every Saturday at CYF we cook and eat together. We share our food and our stories. We learn about each other and the world. We build community.  
+This is everyone's responsibility, so help with what is needed to make this happen, for example, organising the food, setting up the table, washing up, tidying up, etc. You can do something different every week. You don't need to be constantly responsible for the same task.
```

Comment:
```suggestion

This is everyone's responsibility, so help with what is needed to make this happen, for example, organising the food, setting up the table, washing up, tidying up, etc. You can do something different every week. You don't need to be constantly responsible for the same task.
```

Reply:
I was going to add it but I wasn't sure. Thanks for clarifying it :)